packages/avatars/src/styled/StyledAvatar.ts

```diff
@@ -194,8 +194,10 @@ const sizeStyles = (props: IStyledAvatarProps & ThemeProps<DefaultTheme>) => {
 
   return css`
     border-radius: ${borderRadius};
-    width: ${size};
-    height: ${size};
+    /* stylelint-disable declaration-no-important */
+    width: ${size} !important;
+    height: ${size} !important;
```

Comment:
These should get a `/* [1] */` head note to explain why the `!important` is necessary.

Reply:
Updated with a comment